### ISSUE TITLE
Standalone debugger fails to start on Ubuntu 22.04

### ISSUE THREAD

**Issue by adamheinrich** [eclipse-cdt/cdt#256]
Hello, I wanted to update the standalone deugger to the most recent version. I have followed the instructions in https://github.com/eclipse-cdt/cdt/blob/main/StandaloneDebugger.md#try-out-full-eclipse-installation but when I try to launch the debugger using `~/cdtdebugger/cdtdebug.sh`, Eclipse fails with an a message "An error has occured. See the log file.".

I am using Ubuntu 22.04.01 and the version I tried is `eclipse-cpp-2022-12-R-linux-gtk-x86_64.tar.gz`. The output of `java --version` is:

```
openjdk 11.0.17 2022-10-18
OpenJDK Runtime Environment (build 11.0.17+8-post-Ubuntu-1ubuntu222.04)
OpenJDK 64-Bit Server VM (build 11.0.17+8-post-Ubuntu-1ubuntu222.04, mixed mode, sharing)
```

The log file is: [1674380819910.log](https://github.com/eclipse-cdt/cdt/files/10474100/1674380819910.log)

I purged the original 10.0.0 cdtdebugger first (i.e. the folder `~/cdtdebugger` did not exist before running `install.sh`). When instead of running the installed `cdtdebug.sh` I attempt to run eclipse from the unpacked tar, it starts without errors.

Is this a bug or am I doing something wrong? Thanks for any help :)

**Comment by adamheinrich** [comment]
I have just tried to install the Standalone debugger following the same [instructions](https://github.com/eclipse-cdt/cdt/blob/main/StandaloneDebugger.md#try-out-full-eclipse-installation) in a fresh installation of Ubuntu 22.04.01 in Virtualbox and got the same error.

**Comment by jonahgraham** [comment]
> Is this a bug or am I doing something wrong? Thanks for any help :)

This is indeed a bug, thank you for taking the time to report it. It looks like it was broken in 2022-09. The 2022-06 version works and can be used\* still. https://www.eclipse.org/downloads/packages/release/2022-06/r/eclipse-ide-cc-developers


\* Note that there are a few bugs in Ubuntu 22 that you may trip over:

- (fixed in #69) you get an error like `cdtdebugger/cdtdebug.sh: 104: Bad substitution`. Workaround is use bash instead of dash (edit cdtdebug.sh to have `#!/bin/bash`)
- (fixed in #242) CDT does not support dwarf5 which is the default on newest GCC that comes with Ubuntu. See https://github.com/eclipse-cdt/cdt/blob/main/NewAndNoteworthy/CDT-11.0.md#dwarf-versions for more info.
- this issue that you reported which I will try to fix before the pre-release that goes out on Friday.

**Comment by jonahgraham** [comment]
> this issue that you reported which I will try to fix before the pre-release that goes out on Friday.

Fixed now, the 2023-03 M2 build that will be on [this URL](https://www.eclipse.org/downloads/packages/release/2023-03/m2) on Friday looks like it should work fine.
